docker/jenkins-images/Dockerfile-build-base

```diff
@@ -1,17 +1,20 @@
 ARG FROM_VERSION
 FROM harbor.h2o.ai/opsh2oai/h2o-3/dev-r-3.4.1:${FROM_VERSION}
 
-RUN \
-    curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
```

Comment:
This way of installing Nodejs is deprecated and the script will get removed. So I updated the way to install `nodejs==16`.